### PR TITLE
Add source of the node

### PR DESCRIPTION
**Pull request by vikmeup**
Adding the required parameter to specify the source of the node to run.

- if we need manually to run node ourselves
- provide users the ability to run their own in the future, the source would be useful for them to do it

### CODE REVIEW

File: src/interface/TWCoinTypeConfiguration.cpp [hunk 1]
```diff
@@ -304,4 +304,48 @@ TWString *_Nonnull TWCoinTypeConfigurationGetName(enum TWCoinType type) {
...
+    case TWCoinTypeGroestlcoin:
+    case TWCoinTypeZcash:
+    case TWCoinTypeZcoin:
+    case TWCoinTypeViacoin:
+    case TWCoinTypeLux:
+    case TWCoinTypeQtum: string = "https://github.com/trezor/blockbook"; break;
+    case TWCoinTypeICON: string = "https://github.com/icon-project/icon-rpc-server"; break;
+    case TWCoinTypeNEO: string = "https://github.com/neo-project/neo"; break;
+    case TWCoinTypeStellar: string = "https://github.com/stellar/go"; break;
+    case TWCoinTypeEthereum:
```
Comment (armatusmiles) [inline src/interface/TWCoinTypeConfiguration.cpp]: Attention. For all cases below result will be: "https://github.com/ethereum/go-ethereum".
Need add break or empty string

File: src/interface/TWCoinTypeConfiguration.cpp [hunk 2]
```diff
@@ -304,4 +304,48 @@ TWString *_Nonnull TWCoinTypeConfigurationGetName(enum TWCoinType type) {
     return TWStringCreateWithUTF8Bytes(string.c_str());
 }
 
+TWString *_Nullable TWCoinTypeConfigurationGetNodeSource(enum TWCoinType type) {
+    string string;
+    switch (type) {
+    case TWCoinTypeBitcoin:
```
Comment (armatusmiles) [inline src/interface/TWCoinTypeConfiguration.cpp]: Attention. For all cases below result will be: "https://github.com/trezor/blockbook"
Need add break or empty string

**Comment by hewigovens** [comment]
I would prefer to move coin configuration to a json file, what do you think?

**Comment by vikmeup** [comment]
@hewigovens config file makes sense, let's wait for @alejandro-isaza to work on that then. Closing this for now

**Pull request closed without merging**